docs/Concepts/Pruning.md

```diff
@@ -0,0 +1,13 @@
+description: Pruning
+<!--- END of page meta data -->
+
+# Pruning
+
+Use pruning on Mainnet to remove state tries that aren't necessary. Otherwise, the state will continue to grow, and storage will become a problem.
```

Comment:
I think this needs flipping so the the first sentence states *why* you would use pruning and the second sentence describes *what* pruning is.

Reply:
Also, you can use pruning on any network but it's most relevant on Mainnet because it has the largest world state.

Reply:
Updated